source/includes/_endpoint-batch.md

```diff
@@ -62,6 +68,40 @@ The variables IDs must match those of the target dataset since their types will
 }
 ```
 
+#### Managing Append Failures
+
+In case of failures while appending the batch, the dataset will be automatically reverted
+back to the state it was before the append and the batch is automatically deleted.
+
+This behaviour can be changed by providing a `autorollback: false` value into the POST
+performed to append the data. When `autorollback: false` is provided a savepoint to which
+user can revert to is still created, but the dataset is left in errored state for inspection.
+The user can then manually revert to the savepoint and delete the associated batch.
+
+```json
+{
+  "element": "shoji:entity",
+  "autorollback": false,
+  "body": {
+      "source": "<url>"
+  }
+}
+```
+
+In case of appending to an empty dataset you might want to avoid the cost of creating
+a savepoint at all as reverting to it will just lead to an empty dataset which you probably
+just want to delete, in such case a `savepoint: false` value can be provided which will
```

Comment:
yes, `savepoint=false` disable the pre-append savepoint.

In case the pre-append savepoint is disabled `autorollback` is automatically forced to false as rollback can't happen.